azerothcore/azerothcore-wotlk
# fix(Core/Loot): Fixed processing duplicated loot item within different groups.

Fixes #10498



## Issues Addressed:

- Closes #10498
- Closes https://github.com/chromiecraft/chromiecraft/issues/3412

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

`.additem 23162 4`
`.additem 52001 100`
Open all satchels
See the ring has around ~50% drop rate.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
